stdimage/management/commands/rendervariations.py

```diff
@@ -77,9 +89,16 @@ def render(field, images, count, replace, do_render):
 
 def render_field_variations(kwargs):
     kwargs['storage'] = get_storage_class(kwargs['storage'])()
+    igrnore_missing = kwargs.pop('igrnore_missing')
     do_render = kwargs.pop('do_render')
-    if callable(do_render):
-        kwargs.pop('field_class')
-        do_render = do_render(**kwargs)
-    if do_render:
-        render_variations(**kwargs)
+    try:
+        if callable(do_render):
+            kwargs.pop('field_class')
+            do_render = do_render(**kwargs)
+        if do_render:
+            render_variations(**kwargs)
+    except FileNotFoundError as e:
+        if not igrnore_missing:
+            raise
+        else:
+            logger.error(str(e))
```

Comment:
Please don't use the logger here. The command should write to `stderr`.
If you would want to log an exception, you should actually log exception and always provide your own message. The original exception and stack trace will be included anyways. You're log should provide additional information to why the exception was not raised.

Reply:
Anyhow, in this case please write to `self.stderr`.

Reply:
I kinda was too used to trowing everything to the logger without additional thought. Yeah, I will change that *a bit later*.